### ISSUE TITLE
jpg or jpeg images added to winui3 projects do not get the content build action by default.

### ISSUE THREAD

**Issue by hassaanmasood** [microsoft/microsoft-ui-xaml#6437]
### Describe the bug

The jpg format images are not showing when using Image control or when using them as background brushes.  This is a critical bug and should be fixed at the earliest. It is a very basic thing which devs would expect to work.

### Steps to reproduce the bug

1. Create a new WinUI 3 in Desktop app. 
2. Add below xaml inside Stackpanel in MainWindow.xaml
`<Image Source="/Assets/treetops.jpg" Height="100" /> `
3. Add treetops.jpg image (from xaml controls gallery sample app) in assets folder.
4. Run the app.

### Expected behavior

jpg images should be visible correctly. 

### Screenshots

![WinUi jpg bug](https://user-images.githubusercontent.com/58418307/144755858-02efb825-8b1e-435b-8746-b272f77bb241.PNG)
![WinUi jpg bug code](https://user-images.githubusercontent.com/58418307/144755876-c99e3062-9769-4d9b-9449-843764519241.PNG)

### NuGet package version

WinUI 3 - Windows App SDK 1.0 (If you're seeing your issue in older previews of WinUI 3, please try this release)

### Windows app type

- [ ] UWP
- [X] Win32

### Device form factor

Desktop

### Windows version

May 2021 Update (19043)

**Comment by castorix** [comment]
There is no bug
You must add pictures to the project, or they won't be compiled
WinUI 3 is in good shape....

**Comment by hassaanmasood** [comment]
@castorix You are right. The actual problem is when importing files in WinUI Project, the build action of the image is not automatically be set to content whereas, in UWP apps, it is set to content automatically by VS. So I totally forgot about that.

**Comment by castorix** [comment]
> @castorix You are right. The actual problem is when importing files in WinUI Project, the build action of the image is not automatically be set to content whereas, in UWP apps, it is set to content automatically by VS. So I totally forgot about that.

In my tests, Content is set to "Yes" when I import a .jpg in a C++ project
But it is better in a C# project because .jpg are automatically added from the files found in the project directory/subdirectories (mainly in \Assets), no need to import them,

**Comment by StephenLPeters** [comment]
@evelynwu-msft FYI

**Comment by AMArostegui** [comment]
> @castorix You are right. The actual problem is when importing files in WinUI Project, the build action of the image is not automatically be set to content whereas, in UWP apps, it is set to content automatically by VS. So I totally forgot about that.

Same happened to me. I had to manually set the image as 'Content' to make it work. This used to be set automatically in UWP, if I recall correctly.

**Comment by eFail** [comment]
I wasted several hours trying to understand why `<SvgImageSource/>` doesn't work.  No matter what I did, my SVG image would not show up.  Turned out it was because of this bug.  This is a significant deviation from past behavior and should eventually be fixed.  Setting _BuildAction=Content_ fixed my issue instantly.
